basket/urls.py

```diff
@@ -50,3 +55,20 @@
             path("500/", defaults.server_error),
         ]
     )
+
+
+# So django-ninja returns a pydantic validation error in a consistent JSON shape.
+@api.exception_handler(ValidationError)
```

Comment:
would this make more sense to be in the `api.py` file, or does it need to be in `urls.py` for some reason?

Reply:
Yeah that makes sense to me. The `api` object is created here but could probably be imported there or created there and imported here. I'll play with it.